app/src/components/main/modal/modalLayout/modalFooter/modalFooter.jsx

```diff
@@ -102,7 +103,6 @@ export class ModalFooter extends Component {
               </BigButton>
             </div>
           )}
-          {customButton ? <div className={cx('button-container')}>{customButton}</div> : null}
```

Comment:
At the moment we have a case with custom button on the right (in place of the 'ok' button). Here's my suggestion:
1. Add special prop which controls custom button placement:
Modified `customButton` prop should look like this (don't forget to rebase onto fresh `develop` to get this changes):
```
    customButton: PropTypes.oneOfType([
      PropTypes.node,
      PropTypes.shape({
        onClick: PropTypes.func,
        component: PropTypes.func,
        buttonProps: PropTypes.object,,
        left: PropTypes.bool, // <--- this is our new flag
      }),
    ]),
```
2. Add special class to handle custom button position:
```
.left {
  position: absolute;
  left: 0;
}
```
3. Turn on this class depending on this new flag:
```
className={cx('button-container', { left: customButton.left })}
```